docs/cookbook/examples/ami_handler.rst

```diff
@@ -13,7 +13,7 @@ Available as a Gist at https://gist.github.com/3253287
 
 .. seealso::
     Our `support site has more information about Action Menu Items 
-    <https://support.shotgunsoftware.com/hc/en-us/articles/219031318>`_.
+    <https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Developer_dv_shotgrid_dv_custom_amis_html>`_.
```

Comment:
This bring to a 404 page. Should be: https://developer.shotgridsoftware.com/python-api/cookbook/examples/ami_handler.html